docs/content/security.md

```diff
@@ -441,12 +441,12 @@ package system.authz
 
 # client_cns may defined in policy or pushed into OPA as data.
 client_cns := {
-	"my-client": true
+	"my-client-1": true
 }
 
 default allow := false
 
-allow {                                        # Allow request if
+allow {                                    # Allow request if
```

Comment:
[nit] We could do `allow if` with the future keyword and drop the comment. But that's a cleanup that could be another PR, too.

Reply:
That's a better idea, I have updated it to use that instead.